website/blog/2020-03-16-7.9.0.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: post
+title:  "7.9.0 Released: Smaller preset-env output, Typescript 3.8 support and a new JSX transform"
```

Comment:
@hzoo I made the title lowercase (even if I noticed that you changed it to start with upper letters) because:
- The all-lowercase "preset-env" name looked really bead (subjective :stuck_out_tongue:)
- All the other release posts had lowercase words